frontend/src/metabase/dashboard/components/DashCard/DashCard.styled.tsx

```diff
@@ -62,7 +61,7 @@ export const DashboardCardActionsPanel = styled.div<{
   // react-resizable covers panel, we have to override it
   z-index: 2;
   // left align on small cards on the left edge to not make the actions go out of the viewport
-  @container (max-width: 12rem) {
+  @container DashboardCard (max-width: 12rem) {
```

Comment:
I also gave a name to the container, to avoid possible issues with that now that the  container is defined in another file compared to the file using it